R/utils-scenario.R

```diff
@@ -0,0 +1,5 @@
+recode_scenario <- function(x) {
+  out <- gsub("^ipr|^weo", "", x, ignore.case = TRUE)
+  out <- gsub("_", " ", out)
+  tolower(trimws(out))
+}
```

Comment:
This helper aims to recode the values of `scenario` in a more "programatic" way -- trying to avoid using explicit strings and letter-case. The exception here is "ipr" and "weo" -- which we could also eliminate by rewriting the code so that it removes the first word (whatever that is).

If you keep this function do test it because a test suggest we may want something like "ns 2050" but instead we get "ns 2050 2030".